frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog.tsx

```diff
@@ -54,60 +69,83 @@ export const ScheduleMissionDialog = (props: IProps): JSX.Element => {
         timer = setTimeout(() => {
             openPopover()
         }, 300)
-
-        if (sessionStorage.getItem('assetString')) setIsAssetValid(true)
-        else setIsAssetValid(false)
     }
 
     const handleClose = () => {
         clearTimeout(timer)
         closePopover()
     }
 
+    const onClickScheduleMission = () => {
+        setIsScheduleMissionsPressed(true)
+        props.onFrontPageScheduleButtonPress()
```

Comment:
There is some confusing naming here. You click schedule mission and Handle front page schedule button?